app/parse.py

```diff
@@ -8,8 +18,62 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[tag.text for tag in quote_soup.select(".tag")]
+    )
+
+
+def get_next_pages(page_soup: BeautifulSoup) -> str:
+    pagination = page_soup.select_one(".pager")
+
+    if pagination is None or not pagination.select(".next > a"):
+        return None
+
+    next_page = pagination.select_one(".next > a")["href"]
+    next_page_url = urljoin(BASE_URL, next_page)
+    return next_page_url
+
+
+def get_quote() -> [Quote]:
```

Comment:
It's better to use get_quotes name, because it's returning several of them, also for return type `list` use `list(Quote)`